test/Interop/Cxx/class/inheritance/Inputs/functions.h

```diff
@@ -54,7 +54,7 @@ struct Base {
     return i * 2;
   }
 
-  void pure() const __attribute__((pure)) {}
```

Comment:
I think the `const` might not be a problem here. I think the `const` in the clang change might refer to the attribute: https://clang.llvm.org/docs/AttributeReference.html#const

That being said, I am also OK leaving the PR as is.

Reply:
Oh, you're right! Let me bring back `const`.